### DISCUSSION TITLE
Laravel 7 live server - Declaration of TranslatorInterface::setLocale($locale) must be compatible with LocaleAwareInterface::setLocale(string $locale)

### DISCUSSION THREAD

**Discussion by adnandogar** [laravel/framework#33958] [General]
I just installed laravel 7.24 on live server and getting this error on apache error logs:

Declaration of TranslatorInterfacte::setLocale($locale) must be compatible with LocaleAwareInterface::setLocale(string $locale)

Tried many ways to fix this.

Used contracts, transports packages but no luck.

using php 7.3 locally and globally which is same.

how can i fix this?
is there any way to bypass this?

**Comment by devcircus** [comment]
Have you checked other issues here? There are several mentioning the same thing. Looks like it may be a symfony package that needs to be updated.

**Comment by adnandogar** [comment]
I checked issue 
https://github.com/laravel/framework/issues/30655
and 
https://github.com/symfony/symfony/issues/34506
verify php version but does not work a single one.

**Comment by adnandogar** [comment]
this issue got fixed,  thanks
actually the issue is 
CLI version of PHP is different from the version ran in Apache and FastCGI.
this issue can occur with everyone.
